src/gui/accelerators.c

```diff
@@ -1955,7 +1955,7 @@ static void _restore_clicked(GtkButton *button, gpointer user_data)
   gtk_dialog_set_default_response(GTK_DIALOG(dialog), GTK_RESPONSE_REJECT);
 
   GtkContainer *content_area = GTK_CONTAINER(gtk_dialog_get_content_area(GTK_DIALOG (dialog)));
-  GtkWidget *label = gtk_label_new(_("restore default shortcuts\n  or as at startup\n  or when the configuration dialog was opened\n"));
+  GtkWidget *label = gtk_label_new(_("restore default shortcuts\n  either as at startup\n  or when the configuration dialog was opened\n"));
```

Comment:
> There are three different sets of shortcuts that can be restored

To be honest, this is a surprise for me, because the wording of the original phrase was difficult to understand in the way you explained it here. Well, restore _default shortcuts_ (to some state) is not exactly the same as restore shortcuts to _default state_ (as one of the options).

@dterrahe Can we put it like this:
```
Restore shortcuts to one of the possible states:
- default
- as at startup
- as when opening a dialogue
```
?

Reply:
There are many more possible _states_. These are just the ones you can restore _from_. Not _to_, because if you don't have the "clear all newer shortcuts" button checked, you still keep additional shortcuts you may have defined next to the restored ones, so you won't be at the default "state". I assume you look at this text in the context of the dialog it is shown in (which also has three buttons for defaults/startup/edits at the bottom)?

Reply:
Yes, I was looking at this text in the context of the surrounding code. I agree (I just didn't pay attention to it at first) that in this context the prepositions "from" and "to" are not interchangeable, as they often are in other cases.

Did I understand correctly that after this replacement, the text I proposed will be accurate?

May I replace the current text with the following?
```
Restore shortcuts from one of the possible states:
- default
- as at startup
- as when opening a dialog
```

Reply:
If you think it is an improvement. Instead of "the possible" I would say "these".

Reply:
Oh, and instead of "a dialog", say "this dialog".

Reply:
> Instead of "the possible" I would say "these".
> Oh, and instead of "a dialog", say "this dialog".

Thanks for the advice, this is even better. I'll add your changes.

> If you think this is an improvement.

This definitely removes the ambiguity that exists in the original text. The problem is that to convey the correct meaning of the phrase, the word `either` must be in this position: "restore either default shortcuts or ...". So it was possible to correct the phrase in this way, making minimal changes, but I just reworded it to make it look simpler and clearer.